modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsSnapshot.java

```diff
@@ -380,6 +383,8 @@ public ClusterMetricsSnapshot(ClusterGroup p) {
 
             nonHeapCommitted += m.getNonHeapMemoryCommitted();
 
+            addCacheOffHeapAllocatedSize((TcpDiscoveryNode)node);
+
             nonHeapUsed += m.getNonHeapMemoryUsed();
```

Comment:
I would put this line into the new method "addCacheOffHeapAllocatedSize" and rename the latest to "nonHeapMemoryUsage".